### PR TITLE
fix: unbreak KegEditor's SELF display

### PR DESCRIPTION
**Pull request by dchest**
#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui/dev-tools/KegEditor.jsx [hunk 1]
```diff
@@ -147,7 +147,9 @@ function ChatInfo(props) {
         <div className="selectable">
             <Chip className="selectable">{props.c.id}</Chip><br />
             {
-                props.c.allParticipants.map(p => <Chip key={p.username}>{p.username}</Chip>)
+                props.c.allParticipants
```
Comment (anri-asaturov) [inline src/ui/dev-tools/KegEditor.jsx]: Chat#allParticipants is never null, are u sure this fixes something?

Comment (dchest) [reply src/ui/dev-tools/KegEditor.jsx]: Yes, it's not chat when it's self.

Comment (dchest) [reply src/ui/dev-tools/KegEditor.jsx]: Broke here https://github.com/PeerioTechnologies/peerio-desktop/commit/f67bd255402382fb57901fa61b7e94f5a2d4e6e9#diff-cf0c8119d86e68d523c28eee8a68a0b7

Comment (dchest) [reply src/ui/dev-tools/KegEditor.jsx]: I guess the component name is misleading.

**Pull request merged**